eq-author/src/components/CommentsPanel/index.js

```diff
@@ -424,7 +649,7 @@ const CommentsPanel = ({
             Add
           </Button>
         </ButtonGroup>
-      </CommentSection>
+      </CommentAddSection>
     </CommentsPane>
   );
 };
```

Comment:
I think this file is growing quite big. Can we extract many of the components and functionality  here to their own files?

Reply:
removed 580 lines into 4 separate components